locale/en_US/messages.properties

```diff
@@ -0,0 +1,99 @@
+##########################################
+##########################################
+##### Strings visible through the UI #####
+##########################################
+##########################################
+
+##############
+## Homepage ##
+##############
+
+thimbleHeading=Thimble by Mozilla - An online code editor for learners &amp; educators.
+thimbleTagLine=Write the Web <br/> On the Web
+thimbleDescription=Thimble is an online code editor that makes it easy to create and publish your own web pages while learning <a title="Learn about HTML" href="https://developer.mozilla.org/docs/Web/HTML">HTML</a>, <a title="Learn about CSS" href="https://developer.mozilla.org/docs/Web/CSS">CSS</a> &amp; <a title="Learn about JavaScript" href="https://developer.mozilla.org/Learn/JavaScript">JavaScript</a>.
+newProjectBtn=Start a project from scratch
+remixProjectsHeading=Make your own version of these popular projects...
+remixThisProject=Remix this Project
+or=or
+useItToTeach=Use it to teach
+teachingKit=Teaching Kit
+contactEmail=Email Us
+contactTwitter=Talk to us on Twitter
+readTheDocs=Read the Docs
+contribute=Contribute on Github
+termsOfUse=Terms of Use
+privacyPolicy=Privacy Policy
+thimbleFooter=Thimble is part of the Mozilla Learning Network. <br/>Learn to teach like mozilla at <a href="http://teach.mozilla.org">teach.mozilla.org</a>.
+
+# Userbar
+loginWelcome=Hi, {{username}}
+newProjectLink=New Project
+yourProjectsLink=Your Projects
+signOutLink=Sign Out
+loginPrompt=<a href="#" title="Log in to Thimble" id="login-link" data-loginUrl="{{ loginURL }}" class="navbar-login">Log in</a> or <a href="#" title="Create an account" id="signup-link" class="navbar-login">Sign Up</a>
+
+# Remix projects
+remixGoal1=How to design a poster
+remixGoal2=Editing HTML content
+remixGoal3=Editing CSS styles
+remixGoal4=CSS colors
+remixGoal5=Creating an interactive element<br/> with JavaScript
+remixGoal6=Finding Creative Commons-licensed images
+remixGoal7=Navigation inside a website
+remixGoal8=Embedding media in webpages
+remixGoal9=Basic coding ideas
+remixGoal10=Editing JavaScript content
+remixGoal11=Creating an interactive button<br /> with JavaScript
+remixGoal12=Searching for openly licensed<br /> images online
+remixGoal13=Editing for specific word choice
+# "Keep Calm Poster"
+keepCalmPoster=Keep Calm Poster
+keepCalmPosterDescription=Change the design of this poster to make it your own.
+keepCalmPosterTeachingKit=Keep Calm {{ gettext("teachingKit") }}
```

Comment:
I don't think I like this format at all :/ I think we can probably cache `{{ gettext(...) }}` in the template and just have the variable here in instead. So, it will just be like `keepCalmPosterTeachingKit=Keep Calm {{teachingKit}}`